### PR TITLE
global : fix invenio_utils imports

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

File: requirements-devel.txt [hunk 1]
```diff
@@ -37,3 +37,5 @@
 -e git+git://github.com/inveniosoftware/invenio-records.git#egg=invenio-records
 -e git+git://github.com/inveniosoftware/invenio-search.git#egg=invenio-search
 -e git+git://github.com/inveniosoftware/invenio-upgrader.git#egg=invenio-upgrader
+-e git+git://github.com/inveniosoftware/invenio-utils.git#egg=invenio-utils
```
Comment (jirikuncar) [inline requirements-devel.txt]: please keep it sorted

Comment (hachreak) [inline requirements-devel.txt]: done

File: setup.py [hunk 2]
```diff
@@ -41,6 +41,8 @@
     'invenio-records>=0.3.2',
     'invenio-search>=0.1.3',
     'invenio-upgrader>=0.1.0',
+    'invenio-utils>=0.1.1',
```
Comment (jirikuncar) [inline setup.py]: missing `invenio-base`

File: tests/test_tags_restful.py [hunk 3]
```diff
@@ -25,8 +25,8 @@
 from datetime import datetime
 
 from invenio_base.wrappers import lazy_import
-from invenio.ext.restful import validation_errors
-from invenio.ext.restful.utils import APITestCase
+from invenio_ext.restful import validation_errors
+from invenio_ext.restful.utils import APITestCase
 from invenio.testsuite import make_test_suite, run_test_suite
```
Comment (jirikuncar) [inline tests/test_tags_restful.py]: please replace `invenio.testsuite` with `invenio_testing` + `setup.py`

**Comment by jirikuncar** [comment]
ping @hachreak

**Comment by hachreak** [comment]
done

**Comment by jirikuncar** [comment]
@hachreak please rebase and :shipit:

**Pull request closed without merging**